src/software/SfM/main_ExportUndistortedImages.cpp

```diff
@@ -19,14 +20,27 @@ using namespace openMVG::sfm;
 
 #include <stdlib.h>
 
+#ifdef OPENMVG_USE_OPENMP
+#include <omp.h>
+#endif
+
 int main(int argc, char *argv[]) {
 
   CmdLine cmd;
   std::string sSfM_Data_Filename;
   std::string sOutDir = "";
+  bool bExportOnlyReconstructedViews = false;
+#ifdef OPENMVG_USE_OPENMP
+  int iNumThreads = 0;
+#endif
 
   cmd.add( make_option('i', sSfM_Data_Filename, "sfmdata") );
   cmd.add( make_option('o', sOutDir, "outdir") );
+  cmd.add( make_option('r', bExportOnlyReconstructedViews, "exportOnlyReconstructed") );
```

Comment:
perhaps we can use the make_switch -R and check that the command have been used with cmd.used(option)
see https://github.com/openMVG/openMVG/blob/master/src/software/SfM/main_ConvertSfM_DataFormat.cpp#L65